arbnode/dataposter/data_poster.go

```diff
@@ -777,10 +775,16 @@ func (p *DataPoster) Start(ctxIn context.Context) {
 			log.Error("Failed to fetch tx queue contents", "err", err)
 			return minWait
 		}
+		tryToReplace, err := p.redisLock.CouldAcquireLock(ctx)
+		if err != nil {
+			log.Warn("Error checking if we could acquire redis lock", "err", err)
+			// Might as well try, worst case we hit contention on redis and fail
+			tryToReplace = true
+		}
```

Comment:
1) Do we not need to AttemptLock before sendTx / replaceTx?
2) If tryToReplace is false - do we want to continue to the loop below or just exit early?

Reply:
I've moved to entirely removing the redis lock except for anytrust chains where it's necessary to prevent posting two different versions of the same batch to the DAS. The data poster was designed to work well without any sort of locking, because it uses redis pipelines to guarantee atomicity. The batch poster works well too, though I had to update things to eliminate spurious warnings in the logs.